### ISSUE TITLE
Softmax Cross Entropy Classifier

### ISSUE THREAD

**Issue by knieling23** [knieling23/MachineLearningGrp2#2]
- [ ] Forward-Pass mit Gewichtsmatrix und Bias implementieren

 - [ ] Softmax-Funktion für Vorhersagen integrieren

 - [ ] Cross-Entropy-Loss mit L2-Regularisierung berechnen
 
 - [ ] Gradient Descent Optimierung implementieren

 Gradientenberechnung für:

- [ ] Gewichte (mit Regularisierungsanteil)

- [ ] Bias
